cmd/lakefs/cmd/run.go

```diff
@@ -244,6 +246,25 @@ var runCmd = &cobra.Command{
 		if err != nil {
 			logger.WithError(err).Fatal("Emailer has not been properly configured, check the values in sender field")
 		}
+		oidcConfig := cfg.GetAuthOIDCConfiguration()
+		var oauthConfig *oauth2.Config
+		var oidcProvider *oidc.Provider
+		if oidcConfig != nil && oidcConfig.Enabled {
+			oidcProvider, err = oidc.NewProvider(
+				cmd.Context(),
+				oidcConfig.URL,
+			)
+			if err != nil {
+				logger.WithError(err).Fatal("Failed to initialize OIDC provider")
+			}
+			cfg.GetBlockstoreDefaultNamespacePrefix()
+			oauthConfig = &oauth2.Config{
```

Comment:
shouldn't we add the RedirectURL also?

Reply:
Since the redirect URL also contains the host, I construct it ad-hoc in every login.